content/techniques/streaming-files.md

```diff
@@ -0,0 +1,41 @@
+### Streaming Files
+
+There won't always be a time where you're sending back JSON or string responses. There may be times where you'd like to send back a file from the server to the client. In an express application, you may use something like
```

Comment:
```suggestion
There won't always be a time where you're sending back JSON or string responses. There may be times where you'd like to send back a file from the server to the client. In an express application, you may use something like
```

Maybe instead of showing an express example here, we could present how it could be accomplished using the standard, `Response` object (saying it's pretty much the same way as in native express/fastify application). The reason is that we're slowly getting more and more users that are not even familiar with express itself